### ISSUE TITLE
Unable to make TextEdit bookmarks visible

### ISSUE THREAD

**Issue by me2beats** [godotengine/godot#55188]
### Godot version

3.4

### System information

Windows

### Issue description

I can't make `TextEdit` bookmarks visible.

### Steps to reproduce

```gdscript
extends TextEdit

func _ready():
	set_line_as_bookmark(0, true)
```
breakpoint gutter is enabled

**Comment by Calinou** [comment]
You need to enable the bookmark gutter separately from the breakpoint gutter: https://github.com/godotengine/godot/blob/dd3acd74bbd0c682f489693e31961aba54bde10a/scene/gui/text_edit.cpp#L1230-L1238

**Comment by akien-mga** [comment]
Fixed by #55197.
